client/src/pages/Profile.js

```diff
@@ -30,8 +34,12 @@ const areResponsesSuccessful = (...responses) => {
 
 const Profile = () => {
   const { memberID } = useParams();
-  const [isError, setIsError] = useState(false);
-  const [user, setUser] = useState({});
+
+  // Upstream user is the DB version. Local user captures local changes made to the user.
+  const [upstreamUser, setUpstreamUser] = useState({});
```

Comment:
could you explain why we need both upstream and local user state variables? to my understanding you would just need to keep track of the local user changes and then push them to the database, but maybe i'm missing something

Reply:
This is just to implement @Qrtn's idea of disabling the "Submit" button if there are no changes made.